controller/users.go

```diff
@@ -864,6 +864,54 @@ func (c *UsersController) VerifyEmail(ctx *app.VerifyEmailUsersContext) error {
 			return err
 		}
 	}
+
+	if isVerified {
+		verfiedUser := verifiedCode.User
+		tokenEndpoint, err := c.config.GetKeycloakEndpointToken(ctx.RequestData)
+		if err != nil {
+			return errors.NewInternalError(ctx, err)
+		}
+		protectedAccessToken, err := auth.GetProtectedAPIToken(ctx, tokenEndpoint, c.config.GetKeycloakClientID(), c.config.GetKeycloakSecret())
+		if err != nil {
+			log.Error(ctx, map[string]interface{}{
+				"keycloak_client_id": c.config.GetKeycloakClientID(),
+				"token_endpoint":     tokenEndpoint,
+			}, "error generating PAT")
+			// if there's an error, we are not gonna bother the user
+		}
+
+		if protectedAccessToken != "" {
+			// try hitting the admin user endpoint only if getting a PAT
+			// was successful.
+
+			usersEndpoint, err := c.config.GetKeycloakEndpointUsers(ctx.RequestData)
+
+			identity, err := loadKeyCloakIdentity(c.db, verfiedUser)
+			if err != nil {
+				return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, err))
```

Comment:
Log the error but don't return the error. Maybe would be easier to control errors if you move this code which updates KC in a separate method?

Reply:
I logged the keycloak errors without returning it - but this error is a db-related error and I thought it would be serious enough to be reported. What do you think? :)

Reply:
Oh, it’s about loading identity with KC type from our DB. Not about loading it from KC :)
My fault. Let’s return this error then.